exercises/binary_converter.cpp

```diff
@@ -5,3 +5,38 @@
   Insert first number: 8
   The binary number is: 1000
 */
+
+#include <iostream>
+#include <cmath>
+
+using namespace std;
+
+string convert_to_binary(int number, int bits)
+{
+    string binary(bits, '0');
+    
+    for (int i = 0; i < binary.length(); i++)
+    {
+        if (number > 0)
+        {
+            binary[bits - i - 1] = (char)(number % 2) + 48;
```

Comment:
idealmente sarebbe ottimo fare qualcosa come
```
binary += ....number % 2
```
e poi farei una sorta di binary.reverse(), però non ricordo se in C/C++ è immediato farlo

Reply:
Per quanto riguarda l'incremento della stringa forse sarebbe una soluzione più elegrante. Per invertire la stringa, invece,  potrei usare `std::reverse(binary.begin(), binary.end())`. Grazie per la dritta!